.gitignore

```diff
@@ -6,3 +6,4 @@ community
 inventory/host_vars/*/secrets.yml
 *.log
 **/*.retry
+.idea/
```

Comment:
Hey Matt, do you have a global git ignore file as well? If you use that for all your IDE specific files, you don't need to change all projects you are working on. :-)

Reply:
:+1:  https://help.github.com/articles/ignoring-files/#create-a-global-gitignore